client/src/components/MainPage/Spectialization/SpecializationCard/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './style.css';
+
+const Card = (props) => {
+  const { values } = props;
+  return (
+    <ul className="category__box">
+      {values.map((element) => {
```

Comment:
we should validate the existence of values before we map on it to prevent uncaught errors.